Update _index.md (#852)
redis/docs: content/operate/rs/clusters/monitoring/_index.md

---
You can use the metrics that measure the performance of your Redis Enterprise Software clusters, nodes, databases, and shards
to monitor the performance of your databases.

> In the Redis Enterprise Cluster Manager UI, you can see metrics and configure alerts that send notifications based on alert parameters. You can also access metrics and configure alerts through the REST API.

Reviewer: ```suggestion
In the Redis Enterprise Cluster Manager UI, you can view metrics, configure alerts, and send notifications based on alert parameters. You can also access metrics and configure alerts through the REST API.
```

> Redis Enterprise vesion 7.8.2 introduces a new metrics stream engine that exposes new version of prometheus scraping endpoint https://<IP>:8070/v2  (In preview).

Reviewer: ```suggestion
Redis Enterprise version 7.8.2 introduces a preview of the new metrics stream engine that exposes the the v2  Prometheus scraping endpoint at https://<IP>:8070/v2 .
```

> This new engine exports all time-series metrics to external monitoring tools such as Grafana, DataDog, NewRelic and Dynatrace using Prometheus.

Reviewer: ```suggestion
This new engine exports all time-series metrics to external monitoring tools such as Grafana, DataDog, NewRelic, and Dynatrace using Prometheus.
```

The new engine enables real-time monitoring, including full monitoring during maintenance operations, providing full visibility into performance during events such as shards' failovers and scaling operations.

> Users who are already using the existing scraping endpoint for integration, can follow this guide [link] to transition and try the new engine. It is possible to scrape both existing and new endpoints simultaneously, allowing in advanced dashboards preparation and smooth transition when ready.

Reviewer: ```suggestion
If you are already using the existing scraping endpoint for integration, follow this guide [link] to transition and try the new engine. It is possible to scrape both existing and new endpoints simultaneously, allowing advanced dashboard preparation and a smooth transition.
```

To integrate Redis Enterprise metrics into your monitoring environment, see the integration guides for [Prometheus and Grafana]({{< relref "/integrate/prometheus-with-redis-enterprise/" >}}) or [Uptrace]({{< relref "/integrate/uptrace-with-redis-enterprise/" >}}).

Make sure you read the [definition of each metric]({{< relref "/operate/rs/references/metrics/" >}})
so that you understand exactly what it represents.

> ## Cluster Manager Metrics

Reviewer 2: Small style suggestion
```suggestion
## Cluster manager metrics
```
